can/interfaces/pcan/pcan.py

```diff
@@ -88,9 +89,11 @@ def __init__(self, channel, state=BusState.ACTIVE, *args, **kwargs):
 
         """
         if not channel:
```

Comment:
You could just put `channel = 'PCAN_USBBUS1'` in the formal parameter list of `__init__()` and then remove lines 91-94. The same goes for `bitrate` as well AFAIK. We would have to re-add it in the `super().__init__()` call at the end though.